submissions/sofiichuk/frogger-task/app.js

```diff
@@ -0,0 +1,165 @@
+const cell = {
+    width: 101,
+    height: 83
+};
+
+const numberOfRows = 6;
+const numberOfColumns = 5;
+
+const playingField = {
+    width: cell.width * numberOfColumns, // 505
+    height: cell.height * numberOfRows // 498
+};
+
+const character = {
+    width: 60,
+    height: 60
+};
+
+const offsetToTheCenter = 23;
+
+const initialPlayerPosition = {
+    x: 2 * cell.width, // 202
+    y: 5 * cell.height - offsetToTheCenter // 392
+};
+
+const initialEnemyPosition = {
+    x: -2 * cell.width, // -202
+    y: [1, 2, 3].map(rowNumber => rowNumber * cell.height - offsetToTheCenter) // 60, 143, 226
+};
+
+const minSpeed = 100;
+const maxSpeed = 500;
+const enemySpeed = Math.floor(Math.random() * (maxSpeed - minSpeed)) + minSpeed;
+const desynchronizedSpeed = [0, 100, 50].map(speedDelta => enemySpeed + speedDelta);
+
+let count = 0;
+
+// ===========================================================
+
+const score = document.createElement('h3');
+const speed = document.createElement('h4');
+const resetButton = document.createElement('button');
+score.textContent = 'your score is: 0';
+speed.textContent = `game speed is: ${enemySpeed}`;
+resetButton.textContent = 'reset speed';
+resetButton.addEventListener('click', location.reload.bind(location))
+document.body.append(score, resetButton, speed);
+
+// ===========================================================
+
+const Enemy = function (x, y, speed) {
+    this.x = x;
+    this.y = y;
+    this.width = character.width;
+    this.height = character.height;
+    this.speed = speed;
+    this.rival = player;
```

Comment:
`player` is still a global variable.
What blocks you from treating in the same way as you treat speed or coordinates?

Reply:
did that